scrapers/dreamsofspanking.yml

```
@@ -0,0 +1,32 @@
+name: DreamsOfSpanking
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - dreamsofspanking.com/scene/item
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@id="product"]/h3/text()
+      Date:
+        selector: //div[@class="date"]/text()
+        postProcess:
+          - replace:
+            - regex: .*pdated ([0-9]{1,2} [A-z]{3} [0-9]{4})
+              with: $1
+          - parseDate: 2 Jan 2006
+      Details:
+        selector: //div[@class="long-description"]/p/text()
+        concat: "\n\n"
+      Performers:
+        Name: //div[@class="performers"][1]/p/a/text()
+      Tags:
+        Name:
+          selector: //p[@class="tags"]/a/text()
+      Studio:
+        Name: 
+          fixed: Dreams of Spanking
+      Image: //video/@poster
```

```suggestion
      Image: //video/@poster|//img[@class="preview"]/@src
```
The above works for some of the scenes that are missing the video

---

I honestly usually just ignore the "no video" updates. If there is no video there won't be a scene in stash and thus no need to scrape. Or do the scene scrapers also work for galleries?